src/config/configJsonSchema.js

```diff
@@ -232,6 +232,10 @@ module.exports = {
                   required: ['port'],
                   additionalProperties: false,
                 },
+                genesisTime: {
+                  type: ['string', 'null'],
```

Comment:
Actually, this one is always required.

Reply:
Then we have to set a `genesisTime` for `testnet`, even though the tendermint chain isn't running there yet?

Reply:
Platform will be on testnet in a month and half :). Yeah, let’s put something here for now but we will be able to keep config consistent.

Reply:
I think this is resolved now through different handling of the genesis data above?

Reply:
Yep!